domains/sterndev.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "zanemsdev",
+        "email": "r.chevalier.pro@hotmail.com"
+      },
+
+      "record": {
+        "URL": "SterNdev"
```

Comment:
Invalid URL

Reply:
URL = https://sterndev.is-a.dev

Reply:
You cannot use that as a URL as it will create an infinite loop.